InvenTree/templates/InvenTree/settings/plugin_settings.html

```diff
@@ -0,0 +1,112 @@
+{% extends "panel.html" %}
+{% load i18n %}
+{% load inventree_extras %}
+{% load plugin_extras %}
+
+{% block label %}plugin-{{plugin_key}}{% endblock %}
+
+
+{% block heading %}
+{% blocktrans with name=plugin.human_name %}Plugin details for {{name}}{% endblocktrans %}
+{% endblock %}
+
+{% block content %}
+
+<div class="row">
+    <div class="col-md-6">
+        <h4>{% trans "Plugin information" %}</h4>
+        <table class='table table-striped table-condensed'>
+            <col width='25'>
+            <tr>
+                <td></td>
+                <td>{% trans "Name" %}</td>
+                <td>{{ plugin.human_name }}{% include "clip.html" %}</td>
+            </tr>
+            <tr>
+                <td><span class='fas fa-user'></span></span></td>
+                <td>{% trans "Author" %}</td>
+                <td>{{ plugin.author }}{% include "clip.html" %}</td>
+            </tr>
+            <tr>
+                <td><span class='fas fa-calendar-alt'></span></td>
+                <td>{% trans "Date" %}</td>
+                <td>{{ plugin.pub_date }}{% include "clip.html" %}</td>
+            </tr>
+            {% if plugin.version %}
```

Comment:
I think that if there is no version information, we should explicitly say "no version information supplied", rather than hiding this field...